plots-and-tables.ipynb

```diff
@@ -846,7 +846,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.11"
+   "version": "3.7.2"
```

Comment:
This Poetry project requires Python >= 3.8: https://github.com/lpsinger/observing-scenarios-simulations/blob/main/pyproject.toml#L10

Reply:
Hi Leo,
We would like to make a new simulation of the popualtions farah, in its own right.
That is to say, I split from the file farah.h5, so as to obtain the populations farah_bbh, farah_bns and farah_nsbh.

Reply:
Tomorrow I will make a report  about the process and  the result.